R/parliament_data.R

```diff
@@ -29,6 +29,7 @@ parliament_data <- function(election_data = NULL,
                             type = c(
                               "horseshoe",
                               "semicircle",
+                              "thirdcircle",
```

Comment:
Should it be "thirdcircle" or "third_circle"?

Reply:
Could it be both? I'm inclined to allow more synonymous options - I have another branch where I've put in "hemicycle" for "semicircle" since this is a common term in French to describe the format of a such a parliament.